### DISCUSSION TITLE
posts/2024-04-25-microbiome-1/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#13] [Announcements]
# posts/2024-04-25-microbiome-1/

There are trillions of microbes in the air, in the soil, traveling along plant roots, and inside of us.

https://rachel.fast.ai/posts/2024-04-25-microbiome-1/

**Comment by Shuyib** [comment]
Thank for sharing your journey into immunology and study of the microbiome. I particularly like the section on dysbiosis. Which seems to be a big problem for us, and plants. Do you think a core microbiome (mapping out the common microbes not just bacteria in fertile and infertile soils would be good so that we can come up with microbial cocktails like yogurt) would help to change the situation? Would you recommend getting your gut or whole microbiota sequenced from time to time in projects like https://www.griffith.edu.au/griffith-health/clinics/gut-health or should I get my own MinION?https://www.frontiersin.org/articles/10.3389/fcimb.2021.806476/full
